src/Language/Haskell/TH/Syntax/Compat.hs

```diff
@@ -768,6 +770,98 @@ bindSplice_ = bindCode_
 bindSplice_ q c = liftSplice ( q >> examineSplice c)
 # endif
 
+-- $splice
```

Comment:
Nit pick: can you put this Haddock section before the definition of `Splice`? That way, it will be the first thing that someone sees about `Splice`s when viewing the source code, and it will more naturally separate the `Code` parts from the `Splice` parts.

Reply:
yes, good idea, thanks!